templates/blocks/share_buttons.twig

```diff
@@ -1,31 +1,84 @@
-{% set socialLink = social.link ~ '?utm_medium=' ~ utm_medium ~ utm_content_param ~ utm_campaign_param %}
+{% block javascripts %}
+    <script>
+			window.addEventListener('DOMContentLoaded', () => {
+				// If the native share functionality is available on the device, we show the corresponding share buttons.
+				if (!navigator.share) {
+					return;
+				}
+
+				const nativeShareButtons = document.querySelectorAll('.share-buttons .share-btn.native');
+				nativeShareButtons.forEach(nativeShareButton => nativeShareButton.style.display = 'block');
```

Comment:
This results in a small delay before showing the native share button, but I think it's the best way to make sure that the functionality is actually available? 🤔